packages/manager/modules/pci/src/projects/project/ai/apps/add/components/app-attach/app-attach.html

```diff
@@ -2,10 +2,10 @@
 <div class="mb-2">
     <!--Description-->
     <span>
```

Comment:
**nitpick:** Block element inside an inline element is bad. Maybe `span` can be changed to `div`

Reply:
Good point, changed here: https://github.com/ovh/manager/pull/9014/commits/79372a77293dd9c97138604693aedca7c8fd9927